Content.Server/Shuttles/Systems/ShuttleSystem.Impact.cs

```diff
@@ -90,106 +91,114 @@ private void InitializeImpact()
     /// </summary>
     private void OnShuttleCollide(EntityUid uid, ShuttleComponent component, ref StartCollideEvent args)
     {
-        if (TerminatingOrDeleted(uid) || EntityManager.IsQueuedForDeletion(uid)
-            || TerminatingOrDeleted(args.OtherEntity) || EntityManager.IsQueuedForDeletion(args.OtherEntity)
-        )
-            return;
+        //Prevents collision damage if a ship that is colliding or is being collided has a shield.
+        if (TryComp<ShipShieldedComponent>(args.OurEntity, out var ShipShieldedComponent) || TryComp<ShipShieldedComponent>(args.OtherEntity, out var OtherShipShieldedComponent))
 
-        if (!_gridQuery.TryComp(args.OurEntity, out var ourGrid) ||
-            !_gridQuery.TryComp(args.OtherEntity, out var otherGrid)
-        )
             return;
-
-        var ourBody = args.OurBody;
-        var otherBody = args.OtherBody;
-
-        // TODO: Would also be nice to have a continuous sound for scraping.
-        var ourXform = Transform(args.OurEntity);
-        var otherXform = Transform(args.OtherEntity);
-        var worldPoints = args.WorldPoints;
-        var worldNormal = args.WorldNormal;
-
-        for (var i = 0; i < worldPoints.Length; i++)
+        else
```

Comment:
```suggestion
```

Ideally also fix the spacing. The else was entirely unneccesary, if you return directly after an `if` statement then it does the same thing and is so much cleaner. And doesn't fuck the entire Git commit history.